src/containers/EnterpriseIndexPage/EnterpriseIndexPage.test.jsx

```diff
@@ -43,11 +45,12 @@ describe('<EnterpriseIndexPage />', () => {
   it('sets the appropriate props', () => {
     expect(wrapper.props().loading).toEqual(false);
     expect(wrapper.props().error).toEqual(null);
-    expect(wrapper.props().enterprises).toEqual(initialState.enterpriseList.enterprises);
+    expect(wrapper.props().enterprises).toEqual(initialState.table['enterprise-list'].data);
   });
 
-  it('getEnterpriseList dispatches fetchEnterpriseList action', () => {
-    wrapper.props().getEnterpriseList();
+  it('searchEnterpriseList dispatches searchEnterpriseList action', () => {
+    wrapper.props().searchEnterpriseList();
+    // TODO: tohavebeencalledwith?
```

Comment:
Can this comment be more explicit about the reasoning for using `toHaveBeenCalledWith` if it's still a todo?